test/room_test.rb

```diff
@@ -3,22 +3,32 @@
 require './lib/room'
 
 class RoomTest < Minitest::Test
+  def setup
+    @room1 = Room.new(:bedroom, 10, '13')
+    @room2 = Room.new(:living_room, 15, '12')
```

Comment:
since the tests were already created without a setup, it probably cost you more time to move these rooms into a setup than just to add them into the one test that you had to write... it's not a big deal, but do think about only making setup methods when they make sense.